In-house patch to change the default glance image format to
"uar".  This patch is not suitable for pushing upstream.

--- os-client-config-1.14.0/os_client_config/defaults.json.~1~	2016-01-14 17:23:12.571276962 -0800
+++ os-client-config-1.14.0/os_client_config/defaults.json	2016-01-14 17:25:11.564252595 -0800
@@ -11,7 +11,7 @@
   "identity_api_version": "2.0",
   "image_api_use_tasks": false,
   "image_api_version": "2",
-  "image_format": "qcow2",
+  "image_format": "uar",
   "key_manager_api_version": "v1",
   "metering_api_version": "2",
   "network_api_version": "2",
